src/scenes/PostGig.jsx

```
@@ -115,188 +117,268 @@ export const PostGig = () => {
     activeEvent?.id
   ])
 
-  const handlePostSelection = useCallback((option) => {
-    // We pass gameState into resolvePost to allow for complex RNG derivations if needed
-    const gameState = { player, band, social }
-    const result = resolvePost(option, gameState, secureRandom())
+  const handlePostSelection = useCallback(
```

Biome (2.4.4)

[error] This hook does not specify its dependency on currentGig?.venue.

(lint/correctness/useExhaustiveDependencies)